### ISSUE TITLE
Trailing slash in path omits rule.

### ISSUE THREAD

**Issue by mckoss** [firebase/bolt#66]
```
type path /x/y/ { read() = true; }
```
Results in
```
{
  "rules": {
    ".read": "true"
  }
}
```
Expect:
```
{
  "rules": {
    "x": {
      "y": {
        ".read": "true"
      }
    }
  }
}
```

**Comment by mckoss** [comment]
This was parsing as:
```
type path /x/y
/ { read() = true; }
```
(two path statements)

**Comment by mckoss** [comment]
Fixed in v0.4.1
